### ISSUE TITLE
Add ability to filter language list by tags

### ISSUE THREAD

**Issue by Anixias** [proglangdesign/proglangdesign.github.io#187]
Currently, all languages are displayed in alphabetical order. I cannot find a use for tags, despite being able to define them for languages. There should be some simple filter or search mechanism that shows languages that match a given tag list. This would be helpful to attract new users to the languages they are interested in finding.

**Comment by jamestmartin** [comment]
If you'd like to use it, go ahead and use it. If you add tags to `tags.yml` and then add them to languages in `projects.yml`, the feature will be re-enabled automatically.

There was some important bug, but someone will find it and fix it eventually. At this point it's better to just start using the feature than to let it rot forever.

**Comment by fennecdjay** [comment]
is there a list of commonly used/suggested  tags?

**Comment by hg0428** [comment]
> is there a list of commonly used/suggested tags?

I don't know. I'm just going to add a bunch and make a PR.

**Comment by jamestmartin** [comment]
> is there a list of commonly used/suggested tags?

No, because nobody was using the feature until now. I don't think it makes sense to define an ontology without more information on how people think it's important to classify their languages. We can always clean up the tags later, so I think it'd make sense to just add whatever you want until we have more information. (Unless you wanted to try to copy a set of tags from somewhere else.)

That said, I'd suggest considering these criteria for a new tag:

* Would it be useful to filter by this tag? (From the perspective of a language designer *or* programmer.)
* Does this tag have a well-documented meaning? (e.g. Could you provide a link to a secondary source that describes what it means?)
* Is the tag objective? (i.e. not an opinion like "easy", and not relative, like "fast")
* Is the tag a defining characteristic of my language? (e.g. Gwion is musical, but it's not particularly important that it uses braces and semicolons.)
* Could this tag realistically apply to other languages? (i.e. it's not so specific that it could only apply to your language)
* Is the tag specific enough to meaningfully categorize languages? (i.e. it's not so broad that it applies to 90% of languages)
